Doc/library/sched.rst

```diff
@@ -114,9 +113,9 @@ Scheduler Objects
    function passed to the constructor) for the next event, then execute it and so
    on until there are no more scheduled events.
 
-   If *blocking* is false executes the scheduled events due to expire soonest
-   (if any) and then return the deadline of the next scheduled call in the
-   scheduler (if any).
+   If *blocking* evaluates to ``False``, execute the scheduled events due to
```

Comment:
"is false" is commonly used in the documentation to mean "evaluates to False". "is false" sounds correct to me.